src/ex2/PokemonClient.js

```diff
@@ -0,0 +1,66 @@
+export default class PokemonClient {
+  constructor() {
+    this.API_BASE = "https://pokeapi.co/api/v2";
+  }
+
+  async fetchAllPokemons() {
+    try {
+      const pokemons = await fetch(
+        `${this.API_BASE}/pokemon?limit=100000&offset=0`
+      );
+      const pokemonsJson = await pokemons.json();
+
+      return pokemonsJson.results;
+    } catch {
+      (err) => console.log(err);
+    }
+  }
+
+  async fetchPokemonNameById(id) {
+    try {
+      const pokemonsList = await this.fetchAllPokemons();
+      if (pokemonsList[id].name) {
+        return `Catch ${pokemonsList[id].name}`;
```

Comment:
this method should be generic and not return full sentence (Catch xxx), it has 1 purpose which to fetch the desired pokemon and return its name

Reply:
fixed :)